mantle/kola/tests/ignition/journaldEntry.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 Red Hat
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ignition
+
+import (
+	"strconv"
+
+	"github.com/coreos/mantle/kola/cluster"
+	"github.com/coreos/mantle/kola/register"
+	"github.com/coreos/mantle/platform/conf"
+)
+
+func init() {
+	register.RegisterTest(&register.Test{
+		Name:        "coreos.ignition.journald-log",
+		Run:         sendJournaldLog,
+		ClusterSize: 1,
+		UserDataV3: conf.Ignition(`{
+    "ignition": {"version": "3.0.0"},
+    "systemd": {
+        "units": [{
+			"enabled": false,
+			"name": "zincati.service"
+		}]
+    }
+}`),
+	})
+}
+
+func sendJournaldLog(c cluster.TestCluster) {
+	m := c.Machines()[0]
+	out := c.MustSSH(m, "journalctl -o json-pretty | grep -c \"57124006b5c94805b77ce473e92a8aeb\"")
```

Comment:
I think you want `journalctl _MESSAGE_ID=57124006b5c94805b77ce473e92a8aeb` at least instead; then journald will do the filtering, and it's also a *precise* match.

Reply:
See also e.g. https://github.com/coreos/fedora-coreos-config/pull/339/files#diff-6b1b11642c8646b9cd8e6704c1896ae2R27

Reply:
Can you also add a link here to the source code defining the message ID, like that PR does with the message from systemd?

Reply:
Fixed